### ISSUE TITLE
Unblock publication of Georama

### ISSUE THREAD

**Issue by davelab6** [google/fonts#2650]
https://github.com/google/fonts/pull/2578/files was merged, but is blocked from publication:

georama's OS_2 weight class (400) doesn't match with default outline location (100).

This will also effect METADATA

**Comment by RosaWagner** [comment]
Noted.
Questions:
A. I never unblocked a publication: (1.) it consist in merging the modifications in the same PR, or (2) I create a new branch?
B. The fonts have to be (1) reexported with correct usWeightClass, or (2) do you prefer a hotfix?

**Comment by m4rc1e** [comment]
> georama's OS_2 weight class (400) doesn't match with default outline location (100).

We've had this issue before https://github.com/googlefonts/fontbakery/issues/2683. The solution was to set the METADATA.pb weigthClass value to 400. We've done exactly the same thing for Georama so I cannot see why this is being flagged. We also update FB and gftools to do this automatically.

**Comment by thlinard** [comment]
Although the version number is the same, the files of https://github.com/google/fonts/pull/2578 are behind the upstream repo: they don't have the fix for https://github.com/productiontype/Georama/issues/4, while the upstream VFs have it.

**Comment by davelab6** [comment]
https://github.com/google/fonts/blob/e24965e60f784bdfeff869baca4ff94dbb3b542d/ofl/georama/METADATA.pb#L9 has it set to 400

**Comment by davelab6** [comment]
@RosaWagner a. Unblocking publication just means that this seemed ready to go live but got stuck at he last stage, so it's a priority to do the last fixes and get it finished. 1. The pr was merged, so more can't be added there any more. More can be added to the branch that was merged and a new pr made. So you can do as you say in 2 also. 

For bi prefer re-export, but I recommend to always be in touch with upstream before doing any fixing yourself as they might volunteer to fix it themselves

**Comment by RosaWagner** [comment]
Thanks @davelab6. I have 2 questions 

1. What about @m4rc1e comment:
>We've had this issue before googlefonts/fontbakery#2683. The solution was to set the METADATA.pb weigthClass value to 400. We've done exactly the same thing for Georama so I cannot see why this is being flagged. We also update FB and gftools to do this automatically.

2. The variable font contains width instances. So same as Newsreader: is this font special, or I should remove some instances and get in touch with @productiontype about it ?

**Comment by RosaWagner** [comment]
The binaries has also too long PS names (over 31 characters), so I would suggest to reexport it all anyway with contracted family names… Do we have a standard for that by the way ? I have seen this post https://github.com/googlefonts/fontbakery/issues/2179 but it doesn't really set up a standard.

**Comment by productiontype** [comment]
We usually do use short names in our own workflow, but they are known to cause stylelinking issues in some apps. After discussions on NewsReader, it was decided to implement long names in both projects. See commit https://github.com/productiontype/NewsReader/commit/24d0eeefaadaf1274dda86d1155b5545b6a52c1b See discussion short vs long on FB: https://github.com/googlefonts/fontbakery/issues/2179

**Comment by m4rc1e** [comment]
> https://github.com/google/fonts/blob/e24965e60f784bdfeff869baca4ff94dbb3b542d/ofl/georama/METADATA.pb#L9
> 
> has it set to 400

I still don't see how this is an issue.

We currently serve Raleway which has an OS/2 usWeightClass value of 100 (default origin is Thin). While in its metadata, it has [400](https://github.com/google/fonts/blob/master/ofl/raleway/METADATA.pb#L9). Here's an FB issue which discusses the relationship between the METADATA.pb and the OS/2 VF value, https://github.com/googlefonts/fontbakery/issues/2683
